src/app/lib/config/services/afrique.js

```diff
@@ -149,32 +152,37 @@ const service = {
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'Read about our approach to external linking.',
+        text: 'Découvrez notre approche en matière de liens externes.',
       },
       links: [
         {
-          href: 'https://www.bbc.com/news/help-41670342',
-          text: 'Why you can trust the BBC',
+          href: 'https://www.bbc.com/afrique/institutionelles-49283281',
+          text: 'Pourquoi vous pouvez faire confiance à BBC News',
+        },
+        {
+          href: 'https://www.bbc.com/afrique/institutionelles-36826639',
+          text: "Conditions d'utilisation",
         },
         {
-          href: 'https://www.bbc.com/terms',
-          text: 'Terms of Use',
+          href: 'https://www.bbc.com/afrique/institutionelles-36826641',
+          text: 'A propos de la BBC',
         },
         {
-          href: 'https://www.bbc.com/privacy/',
-          text: 'Privacy Policy',
+          href: 'https://www.bbc.com/afrique/institutionelles-36826642',
+          text: 'Politique de confidentialité',
         },
         {
-          href: 'https://www.bbc.com/usingthebbc/cookies/',
+          href:
+            'https://www.bbc.co.uk/privacy/cookies/managing/cookie-settings.html',
```

Comment:
There was just a conversation in Slack saying we can link to `https://www.bbc.co.uk/usingthebbc/cookies/` for cookies rather than `https://www.bbc.co.uk/privacy/cookies/managing/cookie-settings.html` which redirects to the same page.

Reply:
Thank you, changed the urls!